RELEASE_NOTES.md

```diff
@@ -1,3 +1,28 @@
+# Test Kitchen 2.0 Release Notes
+
+Test Kitchen 2.0 is primarily a code cleanup release, but as it does remove functionality upon a limited number of users may have depended, we have chosen to release this as a new major version. We believe that nearly all users will experience no change with this new version, but this release allows us to remove a good amount of legacy logic that will make Test Kitchen a bit faster and certainly easier to maintain going forward.
+
+### Breaking Changes
+
+- Support for the Chef Librarian depsolver has been removed. Users should utilize either Berkshelf or Policyfiles instead.
+- The Chef Solo provisioner no longer supports Chef 10.
+- The Chef-Zero provisioner no longer supports Chef versions prior to 11.8.
+
+### Updated SSH Support
+
+Test Kitchen has been updated to support net-ssh 5, which allows for new SSH key formats. Thanks [@Val](https://github.com/Val) for this fix.
+
+### Other Improvements
+
+- Improvements to log messages better convey what Test Kitchen is actually doing and how to resolve problems
+- The kitchen init command now generates a kitchen.yml with an Ubuntu 18.04 system
+
+# Test Kitchen 1.24.0 Release Notes
+
+## Improved error messages
+
+Test Kitchen now provides improved error messages when it can't find a driver, provisioner, transport, or verifier. It will now output a message listing the plugins that it does know about, which may help users who have simply typed a name in a kitchen.yml file
```

Comment:
```suggestion
Test Kitchen now provides improved error messages when it can't find a driver, provisioner, transport, or verifier. It will now output a message listing the plugins that it does know about, which may help users who have simply typed a name in a `kitchen.yml` file
```

Reply:
Trying to be consistent in formatting of the release notes here.

Reply:
Also, `can't` should be `cannot`.